apps/pygame_sdl2_demo.py

```diff
@@ -136,10 +136,10 @@ def handle_input(self) -> None:
             self.exit_status = 0
             self.running = False
 
-    def run(self):
+    def run(self) -> bool:
         """This is our app main loop"""
         self.running = True
-        self.exit_status = 1
+        self.exit_status = True
```

Comment:
run() returns a boolean right? self.exit_status = 1 > True?

Reply:
exit_status is supposed to be returned to the OS after the program exits.  for example 0 means no error, 1 is an error.  it looks like it was changed to a boolean at some point, but that is a mistake.  it should be an int.  i also see that its not actually passed to the OS, so it must have been lost a while ago.